### PR TITLE
Python Docker

### LINKED ISSUE #6: Python Docker
Install Pyhton on busybox container taht i will run
i have to keep it in sleep and to run more cmd and retrieving the host

### PR DESCRIPTION
**Pull request by ShaindyBorzikowsky**
Using the docker package in Python to create a container and print the host server

Closes #6

### CODE REVIEW

**Comment by bdsoha** [comment]
Please rename the `app` directory to something more significant.
After that, I will begin the code review.

**Comment by ShaindyBorzikowsky** [comment]
Sorry,
I noticed I didn't rename the commit name

**Comment by bdsoha** [comment]
It is important to follow conventions when coding, it makes the overall repo look as if it was developed by a single person, when in fact there were multiple contributors.

- Stick to `kabob-case` for file/folder names.
- Stick to `snake_case` for python files.

**Comment by ShaindyBorzikowsky** [comment]
I changed the folder's name, 
are required more changes?
ש

**Comment by bdsoha** [comment]
**Stick to kabob-case for file/folder names.**

**Comment by ShaindyBorzikowsky** [comment]
Did you mean to **kebab**-case?
I only found information about this

**Comment by ShaindyBorzikowsky** [comment]
If so, I see that a special installation is needed to use stdrename,
And I can't install it.
Maybe it's related to Netfri?

**Comment by bdsoha** [comment]
> If so, I see that a special installation is needed to use stdrename, And I can't install it. Maybe it's related to Netfri?

Why do you need any installations.
Where are your files, on your PC?

**Comment by ShaindyBorzikowsky** [comment]
Yes

**Comment by bdsoha** [comment]
> Yes

So change the file name in the UI

**Comment by ShaindyBorzikowsky** [comment]
Now it's OK?

**Comment by bdsoha** [comment]
No.
**Please read the instructions carefully**

It is important to follow conventions when coding, it makes the overall repo look as if it was developed by a single person, when in fact there were multiple contributors.

Stick to `kebob-case` for file/folder names.
Stick to `snake_case` for **python** files.

File: dockerOnPython/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+```bash
+pip install --trusted-host pypi.org --trusted-host pypi.python.org --trusted-host files.pythonhosted.org docker
+```
+If you are running behind a Netfri security certificate, you may need to add additional --trusted-host flags for your internal package repository.
+
+Next, run the Python script using the following command:
+
+```bash
+python dockerPython.py
```
Comment (bdsoha) [inline dockerOnPython/README.md:25]:
```suggestion
python docker_python.py
```

File: dockerOnPython/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Next, run the Python script using the following command:
+
+```bash
+python docker_python.py
+```
+
+The Python script will create a Docker container running the busybox image and running the command sleep 3600. The container will run in the background, and its hostname will be printed to the console.
+
+Note: The purpose of this script is to demonstrate how to use the Docker Python library, and the specific command (sleep 3600) is used simply as an example. You can modify this command to run any container or command that you like.
```
Comment (bdsoha) [inline dockerOnPython/README.md:30]:
```suggestion
Note: The purpose of this script is to demonstrate how to use the Docker Python library, and the specific command _(sleep 3600)_ is used simply as an example. You can modify this command to run any container or command that you like.
```

**Comment by bdsoha** [comment]
**🎉 Congrats on your first merged PR.**

You will soon be added as a collaborator to this repo.
In the meantime, please prepare a short presentation *(in English)* describing:

- The task in general.
- Things you learned along the way.
- Tips for other team-members for future tasks.

File: dockerOnPython/README.md (hunk 2 above)
Comment (ShaindyBorzikowsky) [reply dockerOnPython/README.md]: Why is this change needed?
I only want to understand

Comment (bdsoha) [reply dockerOnPython/README.md]: It makes the code *italic*. Helps with readability. 
Usually, parenthesis in text are used to indicate secondary information. Italic blocks are cognitively interpreted as secondary.

**Comment by bdsoha** [comment]
You have been sent an email invitation to the [SkyVar Organization](https://github.com/skyvar).
Please open your cooperate email and finish the registration process.

**Pull request merged**